FccFullAnalysis.md

```diff
@@ -158,7 +159,7 @@ The **gen-level** and **reco-level** output trees can be found here:
 Download the python code:
 
 ```
-git clone git@github.com:selvaggi/tutorials.git
+git clone https://github.com/selvaggi/tutorials.git
```

Comment:
What is the plan with those? Do we want to put these example scripts into one of our repositories? E.g. fcc-physics?